source/beamline/IDEAS/g13ElementDetailedDetectorView.cpp

```diff
@@ -0,0 +1,72 @@
+#include "g13ElementDetailedDetectorView.h"
```

Comment:
You are missing the IDEAS prefix here too.  Also, classes don't start with lower case letters.  You should also include the "e" for the germanium detector so that it is explicit (i.e.: IDEASGe13ElementDetailedDetectorView).